pkgs/applications/science/math/R/default.nix

```diff
@@ -2,7 +2,10 @@
 , libjpeg, libpng, libtiff, ncurses, pango, pcre, perl, readline, tcl
 , texLive, tk, xz, zlib, less, texinfo, graphviz, icu, pkgconfig, bison
 , imake, which, jdk, openblas
+, tzdata
 , withRecommendedPackages ? true
+
```

Comment:
Adding an empty line within function argument lists like this is very uncommon in Nixpkgs.